web/app/themes/dp-timber-theme/views/html-header.twig

```diff
@@ -12,4 +12,5 @@
     <link rel="author" href="{{ site.theme.link }}/humans.txt" />
     <link rel="pingback" href="{{ site.pingback_url }}" />
     <link rel="profile" href="http://gmpg.org/xfn/11">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/css/bootstrap.min.css" rel="stylesheet">
```

Comment:
Do we need this anymore as we are adding this in different way.

Reply:
Yes, I will remove it